### ISSUE TITLE
library.properties using old URL?

### ISSUE THREAD

**Issue by owennewo** [simplefoc/Arduino-FOC#12]
Referring to:
https://github.com/simplefoc/Arduino-FOC/blob/master/library.properties
The url is
```
url=http://askuric.github.io/Arduino-FOC
```
I think you've moved the repo to simplefoc organization so has the URL changed??

In platformio I'm only able to install up to 1.3.0 which I'm guessing is when you cut over to the new git repo??

On a small but related note the space in library name "Simple FOC" is a minor inconvenience in platformio/vscode.  If I have an error in compiler terminal, I can't click it to go to line number because of the space in library name (and thus space in path).  I imagine changing this is risky because you'd be left with too libraries with similar names.  So maybe best to do nothing about this nitpic.

**Comment by owennewo** [comment]
A picture paints a thousabd words...
![image](https://user-images.githubusercontent.com/15650427/91566416-8e6be780-e93b-11ea-807e-4739d1f4b131.png)

My workaround is to have this in my platformio.ini:
```
[env:esp32doit-devkit-v1]
platform = espressif32
board = esp32doit-devkit-v1
framework = arduino
monitor_speed = 115200
lib_deps = 
    https://github.com/simplefoc/Arduino-FOC
    SPI
    Wire
```

**Comment by askuric** [comment]
You are right, it is the old docs website. I will change it right away. 

Regarding the platformio, I did not use it so far and I am not sure why does this happen. But it can be that the reason is because I changed the repo location from my personal git to the simplefoc git.  (old docs webiste has nothing to do with changing the repo location)
All the links should still redirect to the new repository for example:
https://github.com/askuric/Arduino-FOC 
should still link to:
https://github.com/simplefoc/Arduino-FOC

To be honest I thought that platformio just copies Arduino libraries but it seems that it has it's own mechanism to retrieve them. 

Regarding the name, I have to look into it. I know it is strange to have this space, but on the begging I was not thinking about it a lot. And I have seen that there is a lot of Arduino libraries with the spaces in names so I just left it there. I have to see if that would create some dependency problems, at the moment I have no idea.
